examples/node/MarketDataSubscription_LongPoll.js

```diff
@@ -75,4 +82,6 @@ function unsubscribe()
     });    
 }
 
+var opts = cmdline.parse({host: 'http-api-host', port: 3000});
```

Comment:
Since the whole purpose of using `cmd` is to eliminate the need for users to actually look into the code, I don't think having `default` value for mandatory fields are necessary. My take would be only providing default values for optional parameters(none for now), and generating help docs if any mandatory fields are missing. `nomnom` has `required` options that provides this feature.